shared/i18n/locales/en_US/translation.json

```diff
@@ -326,11 +327,20 @@
   "We were unable to load the document while offline.": "We were unable to load the document while offline.",
   "Your account has been suspended": "Your account has been suspended",
   "A team admin (<em>{{ suspendedContactEmail }}</em>) has suspended your account. To re-activate your account, please reach out to them directly.": "A team admin (<em>{{ suspendedContactEmail }}</em>) has suspended your account. To re-activate your account, please reach out to them directly.",
+  "Are you sure about that? Deleting the <em>{{groupName}}</em> group will cause its members to lose access to collections and documents that it is associated with.": "Are you sure about that? Deleting the <em>{{groupName}}</em> group will cause its members to lose access to collections and documents that it is associated with.",
+  "You can edit the name of this group at any time, however doing so too often might confuse your team mates.": "You can edit the name of this group at any time, however doing so too often might confuse your team mates.",
   "{{userName}} was added to the group": "{{userName}} was added to the group",
   "Add team members below to give them access to the group. Need to add someone who’s not yet on the team yet?": "Add team members below to give them access to the group. Need to add someone who’s not yet on the team yet?",
   "Invite them to {{teamName}}": "Invite them to {{teamName}}",
   "{{userName}} was removed from the group": "{{userName}} was removed from the group",
+  "Add and remove team members in the <em>{{groupName}}</em>\n              group. Adding people to the group will give them access to any\n              collections this group has been added to.": "Add and remove team members in the <em>{{groupName}}</em>\n              group. Adding people to the group will give them access to any\n              collections this group has been added to.",
```

Comment:
One last thing, could we remove these newlines? While they are okay in the translation strings when necessary, we should avoid them when they have no semantic value and are just an artifact of the code.

Reply:
I will definitely. It has newlines because the length of the line in the file gets too long otherwise.